server.js

```diff
@@ -26,6 +26,7 @@ app.use(bodyParser.urlencoded({extended: true}));
 
 app.get('/api/comments', function(req, res) {
   fs.readFile(COMMENTS_FILE, function(err, data) {
+    if (err) console.log(err);
```

Comment:
Please use blocks (with curlies). And perhaps uses `console.error`. We should perhaps actually handle the error though. Otherwise we might just get another error when we call `JSON.parse`.

If we're going to say we're handling the error, let's handle them all and not just log.

Personally I don't care terribly - these are only meant to be *good enough*, not production ready. If you'd like to make this versatile, feel free.